OpenTelemetry.proj

```diff
@@ -7,12 +7,15 @@
     <!-- Not pack SemanticConventions project for now -->
     <SolutionProjects Remove="src\OpenTelemetry.SemanticConventions\OpenTelemetry.SemanticConventions.csproj" />
     <PackProjects Remove="src\OpenTelemetry.SemanticConventions\OpenTelemetry.SemanticConventions.csproj" />
+  </ItemGroup>
 
+  <ItemGroup Condition="'$(RunningDotNetPack)' == 'true'">
     <!-- Skip building these projects when publish packages workflow runs as these projects need to depend upon instrumentation libraries taking a project reference
     instead of  package reference on API/SDK-->
-    <SolutionProjects Remove="examples\**\*.csproj" Condition="'$(RunningDotNetPack)' == 'true'"/>
-    <SolutionProjects Remove="test\Benchmarks\Benchmarks.csproj" Condition="'$(RunningDotNetPack)' == 'true'"/>
-
+    <SolutionProjects Remove="docs\**\**\*.csproj" />
+    <SolutionProjects Remove="examples\**\*.csproj" />
+    <SolutionProjects Remove="test\Benchmarks\Benchmarks.csproj" />
```

Comment:
We could actually just skip building the test projects altogether when running the publish package workflow. For now, I would like to just get this PR merged to unblock `1.7.0-alpha.1` release.